dbt/logger.py

```diff
@@ -33,6 +33,14 @@
 logger.setLevel(logging.DEBUG)
 logging.getLogger().setLevel(logging.CRITICAL)
 
+# Quiet these down in the logs
+logging.getLogger('botocore').setLevel(logging.INFO)
+logging.getLogger('requests').setLevel(logging.INFO)
+logging.getLogger('urllib3').setLevel(logging.INFO)
+logging.getLogger('google').setLevel(logging.INFO)
+logging.getLogger('snowflake.connector').setLevel(logging.INFO)
+logging.getLogger('parsedatetime').setLevel(logging.INFO)
```

Comment:
@drewbanin i'm confused as to how these got in here...?

Reply:
i tried to quiet down the verbosity of logs in the `logs/dbt.log` file. Tristan saw debug longs in his console -- I wonder if that's related?